_posts/2024-05-14-ASUworkshop.md

```diff
@@ -0,0 +1,169 @@
+
+This two-day workshop is intended to transform attendees from users to
+developers and will cover an introduction to MDAnalysis, software best
+practices, and guide participants through the creation of their own [MDAKit][mdakits].
+
+### Workshop outline
+
+MDAnalysis developers and experienced MolSSI instructors will teach
+participants how to build their own MDAnalysis-based Python packages by guiding
+them through the development of their own MDAKits. Specifically, the workshop
+will include 3 modules: (1) an introduction to using and writing custom
+analyses in MDAnalysis; (2) an overview of software development and maintenance
+best practices; and (3) an interactive hackathon session where participants
+write their own MDAKits.
+
+The program will run from the morning of Monday, June 24th until the early
+evening of Tuesday June 25th. 
+
+Parts of the workshop will be held in a hybrid mode for those unable to attend
+in-person. Registration for the hybrid components is required. Recordings of
+hybrid sessions will be made available after the workshop.
+
+Breakfast and lunch will will be provided together with tea/coffee during
+breaks.
+
+
+
+### Registration
+
+Attendance at this workshop will be *free*. 
+
+A number of **bursaries** will be available to assist with travel and
+accommodation costs for participants from outside the Phoenix area. Bursaries
+(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
+accommodation* (single room within 5 minutes of the workshop venue) will be
+awarded.
+
+#### Application with bursary request (non-local attendees)
+
+Register by **May 27, 2024** (midnight anywhere on Earth).
+
+We especially welcome your application if you consider yourself being part of
+an under-represented demographic in the computational molecular sciences.
+
+Bursaries are reserved to enable the attendance of participants from outside
+the greater Phoenix area.
+
+Note that airfare for travel awardees will have to be booked by June 3 through
+our travel agency so as an awardee you will have to be able to respond
+immediately and act quickly.
+
+
+#### Application without bursary request or local attendees
+
+Register by **June 5, 2024** (midnight anywhere on Earth).
+
+
+### Travel
+
+[Tempe][] is part of the "Valley of the Sun" metropolitan area, which encompasses
+[Phoenix][], the 5th most populous city in the US. Today the area is home to more
+than 4.8 million people and encompasses the ancient lands of the Akimel O'odham
+(Pima), Maricopa, and Tohono O'odham tribes.
+
+
+#### Airport
+
+**Phoenix Sky Harbor** [PHX][] airport has direct connections to all major hubs
+in the US.
+
+PHX airport is 5.5 miles (10 minutes by car/rideshar) from the workshop venue.
+
+#### Car/rideshare
+
+To get from the airport to Tempe either take a cab (traditional or self-driving
+Waymo) or rideshare (Lyft, Uber).
+
+However, public transport works quite well, too.
```

Comment:
I'm not used to the US, but shouldn't public transports be the first suggestion/choice?

Reply:
> I'm not used to the US

Europe has entered the chat :)

I do agree that if public transport is safe & effective it should be listed first - although I also know that this isn't always the case everywhere.

Reply:
Generally I agree with you and I re-arranged.

Public transport in Phoenix is not good _except_ when going along the Light Rail. @ianmkenney does have some interesting storied about encounters on the Light Rail at night but generally I'd consider it reasonably safe. If your destination is close to the Light Rail then use it, otherwise I'd take a car.

Reply:
That's fair. If they are not particularly safe or effective, then it's fine to list alternative first.